src/main/webapp/startup.js

```diff
@@ -0,0 +1,12 @@
+/*
+** Empty the event list and add the current events
```

Comment:
Only empties the list for now. Add a "TODO(remusn):" Where the other behaviour will go.